controllers/vspherecluster_controller.go

```diff
@@ -125,6 +126,13 @@ func (r *VSphereClusterReconciler) reconcileDelete(ctx *context.ClusterContext)
 func (r *VSphereClusterReconciler) reconcileNormal(ctx *context.ClusterContext) (reconcile.Result, error) {
 	ctx.Logger.Info("Reconciling VSphereCluster")
 
+	// TODO(akutz) Update this logic to include infrastructure prep such as:
+	//   * Downloading OVAs into the content library for any machines that
+	//     use them.
+	//   * Create any load balancers for VMC on AWS, etc.
+	ctx.VSphereCluster.Status.Ready = true
```

Comment:
I guess this will change for #538 as it makes capv provision infrastructure

Reply:
Yes, maybe. Although I don't know how we can have two reconcilers influencing the infrastructure ready status. Need to discuss.

Reply:
> I don't know how

I know _technically_. I mean from a design perspective I'm not sure how to do this in a way that works. It's early. I may be having trouble conceptualizing it :)

Reply:
ack